### PR TITLE
Use rel=noopener on user-defined links that open in new tab

### PR DESCRIPTION
**Pull request by zachmullen**
@girder/developers PTAL at this security fix. https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

### CODE REVIEW

**Comment by danlamanna** [comment]
Do we need to add `noreferrer` (for FF support)?

**Comment by cpatrick** [comment]
yes

**Comment by zachmullen** [comment]
Done

**Comment by cpatrick** [comment]
LGTMWTIH.

File: clients/web/src/templates/widgets/itemList.jade [hunk 1]
```diff
@@ -2,16 +2,16 @@ ul.g-item-list
   each item in items
     li.g-item-list-entry
       if checkboxes
-        input.g-list-checkbox(type="checkbox", g-item-cid="#{item.cid}")
-      a.g-item-list-link.g-right-border(g-item-cid="#{item.cid}")
+        input.g-list-checkbox(type="checkbox", g-item-cid=item.cid)
+      a.g-item-list-link.g-right-border(g-item-cid=item.cid)
         i.icon-doc-text-inv
-        | #{item.get('name')}
```
Comment (msmolens) [inline clients/web/src/templates/widgets/itemList.jade]: This, and the other unrelated changes, should be part of a separate commit, or even a separate PR.

**Comment by msmolens** [comment]
Add a description of the security problem to the commit message.

**Comment by danlamanna** [comment]
I found a number of other results when grepping for `_blank` - is there a reason those are ignored?

**Comment by zachmullen** [comment]
I was only targeting user-defined data, rather than static links. Are there any of those that I missed?

**Comment by danlamanna** [comment]
@zachmullen Nope :)

**Comment by jcfr** [comment]
Is there a way to provide a jade mixin or similar to encapsulate this ?

**Comment by zachmullen** [comment]
Backed out the style improvements.
